### PR TITLE
Metrics for GRPC server

### PR DESCRIPTION
**Pull request by cv65kr**
Follow up PR: https://github.com/roadrunner-server/grpc/pull/62

### CODE REVIEW

File: docs/en/app-server/grpc.md [hunk 1]
```diff
@@ -138,6 +138,14 @@ grpc:
     destroy_timeout: 60s
 ```
 
+### Emitted metrics
```
Comment (rustatian) [inline docs/en/app-server/grpc.md:141]: Small clarification, `gRPC` plugin also provides standard worker metrics: https://github.com/roadrunner-server/sdk/blob/master/metrics/metrics.go, could you please add them as well? 
- Total memory for all workers
- Per worker state
- Per worker memory
- Total number of workers
- Ready workers
- Working workers
- Invalid workers

Comment (cv65kr) [reply docs/en/app-server/grpc.md:141]: I thought that are common metrics for plugins with workers and I should be stick only to these related with GRPC server in documentation. No problem I will update it tomorrow.

Comment (cv65kr) [reply docs/en/app-server/grpc.md:141]: Updated

**Review by rustatian — APPROVED** [review]
lgtm, thanks 👍🏻

**Pull request merged**